src/style.css

```diff
@@ -1 +1,125 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
+  padding: 0;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search--form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search--form__big {
+  width: 100%;
+  height: 70px;
+}
+
+.search--form__small {
+  width: 100%;
+  height: 42px;
+}
+
+.search--form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search--form__label::after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+.search--form__input::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+.search--form__input_big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search--form__input_small::placeholder {
+  font-size: 14px;
+  line-height: 22px;
+}
+
+.search--form__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.search--form__input_big {
+  margin-bottom: 20px;
```

Comment:
Avoid fixing container sizes like heights unless it's a specific requirement. Let the content size dictate the container size to avoid overflow or accidental scroll bars.